_includes/sections/skills.html

```diff
@@ -0,0 +1,43 @@
+<div class="container">
+    <div class="row">
+        <div class="col-lg-12 text-center" id="i18_skills">
+            <div class="navy-line"></div>
+            <h1><span data-i18n="skills.my_skills">My Skills</span></h1>
+        </div>
+    </div>
+    <div class="row features-block">
+        <div class="wow zoomIn col-lg-6 col-lg-offset-3">
+            <canvas id="{{ site.data.index.skills.id }}" height="500" width="500"></canvas>
+        </div>
+        <div class="col-lg-1"></div>
+        <script>
+        var ctx = document.getElementById("{{ site.data.index.skills.id }}");
```

Comment:
I'm not sure, whether we will ever use this file. It is just preserved as part of template, but will probably disappear.